src/components/RemoveComponent.js

```diff
@@ -7,7 +7,7 @@ const RemoveComponent = (props) => {
   const { readOnly, removeComponent, onRemove, className, tag, index } = props;
 
   const onKeydown = (event) => {
-    if (event.keyCode === KEYS.ENTER || event.keyCode === KEYS.SPACE) {
+    if (KEYS.ENTER.indexOf(event.keyCode) !== -1 || event.keyCode === KEYS.SPACE) {
```

Comment:
```suggestion
    if (KEYS.ENTER.includes(event.keyCode) || event.keyCode === KEYS.SPACE) {
```
for better readability as it returns boolean